src/mm-datepicker/doc.json

```diff
@@ -0,0 +1,244 @@
+			"type":"Date",
+			"description":"In dual configuration this is the selected start date. In single configuration this is the only date available.",
+			"default":"null"
+		},		
+		{
+			"name":"startEnabled",
+			"type":"boolean",
+			"description":"Default value for the enable/disable checkbox which appears when startEnabledLabel is specified",
+			"default":""
+		},
+		{
+			"name":"startUserEnableable",
+			"type":"boolean",
+			"description":"Determines if the user is able to control the start enabled checkbox state.",
+			"default":"true"
+		},		
+		{
+			"name":"startEnabledLabel",
+			"type":"string",
+			"description":"A label used for the start checkbox. Typically this is used for scenarios where there is a 'parent' date that the picker must constrain to",
+			"default":"null"
+		},		
+		{
+			"name":"startLabel",
+			"type":"string",
+			"description":"The label for the lefthand section of the picker when in dual mode. Typically this is the start date.",
+			"default":"start"
+		},		
+		{
+			"name":"startDate",
+			"type":"string",
+			"description":"A string in the dateformat used for the default",
+			"default":"null"
+		},		
+		{
+			"name":"startTime",
+			"type":"Date",
+			"description":"A date object representing the default time to use in the time input.",
+			"default":"12:00"
+		},		
+		{
+			"name":"startTimePeriod",
+			"type":"string",
+			"description":"The meridian value to default the meridian dropdown to",
+			"default":"am"
+		},		
+				{
+			"name":"end",
+			"type":"Date",
+			"description":"In dual configuration this is the selected end date. In single configuration this is the only date available.",
+			"default":"null"
+		},		
+		{
+			"name":"endEnabled",
+			"type":"boolean",
+			"description":"Default value for the enable/disable checkbox which appears when endEnabledLabel is specified",
+			"default":""
+		},
+		{
+			"name":"endUserEnabled",
+			"type":"boolean",
+			"description":"Determines if the user is able to control the end enabled checkbox state.",
+			"default":"true"
+		},	
+		{
+			"name":"endEnabledLabel",
+			"type":"string",
+			"description":"A label used for the end checkbox. Typically this is used for scenarios where there is a 'parent' date that the picker must constrain to",
+			"default":"null"
+		},		
+		{
+			"name":"endLabel",
+			"type":"string",
+			"description":"The label for the lefthand section of the picker when in dual mode. Typically this is the end date.",
+			"default":"end"
+		},		
+		{
+			"name":"endDate",
+			"type":"string",
+			"description":"A string in the dateformat used for the default",
+			"default":"null"
+		},		
+		{
+			"name":"endTime",
+			"type":"Date",
+			"description":"A date object representing the default time to use in the time input.",
+			"default":"12:00"
+		},		
+		{
+			"name":"endTimePeriod",
+			"type":"string",
+			"description":"The meridian value to default the meridian dropdown to",
+			"default":"am"
+		}	
+	],
+	"events": [
```

Comment:
May want to document the events - as end users will probably want to update their UI on "close" etc...